### DISCUSSION TITLE
Что такое неймспейсы и области видимости

### DISCUSSION THREAD

**Discussion by pingvin1308** [cleannetcode/CsharpForBeginers#4] [Новички]
Продолжить или сделать свою реализацию игры HuntTheWumpus

Сюда можно будет оставлять свои ссылки на решенную домашку. А также тут будет обсуждение ваших домашних заданий. Хочется верить, что мы здесь будем активно помогать друг другу)

**Comment by 8bit-com** [comment]
https://github.com/8bit-com/Wumpus.git начал делать Wumpus в функциональном стиле
